### ISSUE TITLE
Avatar image looks different on desktop and mobile (different cropping)

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#2395]
Version: Version: 2.1.2-alpha.14 (but it's not a new issue, we've just spotted it recently)
System: all

Issue: Vertical identicon looks different on desktop and mobile, and the border radius is different. It's not just a matter of rounding, because as you can see below, quite a significant portion of the identicon is cut on mobile, not just on corners—look at the blue stripe that is not present on the mobile identicon at all. It happens only when the user upload picture that is higher than longer (which looks horrible regardless of the platform). 

<img width="206" alt="315281487-c70a0410-0da7-40d2-8b58-be8fb4105be9" src="https://github.com/TryQuiet/quiet/assets/68909168/266f8471-7ba3-423f-b421-1166348af57e">
<img width="142" alt="315281606-de8225fd-dd6f-408d-99a5-63831da9c1e5" src="https://github.com/TryQuiet/quiet/assets/68909168/4aeb5ca7-41e7-4af5-8ac7-8f269949e946">

**Comment by holmesworcester** [comment]
Changed the title because 'identicon' refers to the placeholder image not the user-uploaded image.

The solution here is to have a real cropping interface for non-square photos.

**Comment by kingalg** [comment]
Yes to both, @holmesworcester. I was checking the identicon issue a few minutes earlier, and it stuck in my head. We also definitely need some cropping and, in general, better handling of avatars.
